NAACCR/manual work/manual_work_08-Feb-23.sql

```diff
@@ -0,0 +1,171 @@
+--It is slightly modified version of https://github.com/OHDSI/Vocabulary-v5.0/issues/740
+TRUNCATE concept_relationship_manual_refresh;
+CREATE TABLE concept_relationship_manual_refresh
+(
+    concept_code_1   varchar(50),
+    concept_code_2   varchar(50),
+    vocabulary_id_1  varchar(20),
+    vocabulary_id_2  varchar(20),
+    relationship_id  varchar(20),
+    valid_start_date DATE,
+    valid_end_date   DATE,
+    invalid_reason   varchar(1)
+)
+;
+
+UPDATE concept_relationship_manual_refresh
+SET valid_start_date= CURRENT_DATE-1;
+
+
+--CM population
+TRUNCATE concept_manual;
+--CM population
+INSERT INTO concept_manual (concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept, concept_code, valid_start_date, valid_end_date, invalid_reason)
+SELECT concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept, concept_code, valid_start_date, valid_end_date, invalid_reason
+FROM concept_manual_backup_2023_02_08
+;
+
+--CHECK THE # OF CODES OVERLAPPING BETWEEN MANUAL REFRESH AND PREVIOUS CRM version in case of identical relationships and thier validities
+SELECT *
+FROM concept_relationship_manual_backup_2023_02_08 crmb
+WHERE EXISTS (
+    SELECT 1
+    from concept_relationship_manual_refresh crmr
+    WHERE crmr.concept_code_1=crmb.concept_code_1
+    and crmr.vocabulary_id_1=crmb.vocabulary_id_1
+    and crmr.relationship_id=crmb.relationship_id
+          )
+and crmb.invalid_reason IS NULL
+and crmb.concept_code_1<>crmb.concept_code_2
+and crmb.vocabulary_id_1<>crmb.vocabulary_id_2
+;
+
+--1toM check
+-- 334 rows including simultaneous maps to Descendant and Ancestor- TBD
+--Clinically relevant
+SELECT DISTINCT *
+   FROM concept_relationship_manual_refresh
+JOIN concept c ON concept_relationship_manual_refresh.concept_code_2 = c.concept_code
+and c.vocabulary_id='Cancer Modifier'
+    WHERE (concept_code_1,relationship_id) IN (
+        SELECT concept_code_1,relationship_id
+   FROM concept_relationship_manual_refresh
+   GROUP BY concept_code_1,relationship_id HAVING count(DISTINCT concept_code_2)>1
+        )
+;
+
+
+
+
+--CRM process
+TRUNCATE concept_relationship_manual;
+INSERT INTO concept_relationship_manual (concept_code_1, vocabulary_id_1, relationship_id, valid_start_date,
+                                         valid_end_date, invalid_reason, concept_code_2, vocabulary_id_2)
+SELECT DISTINCT
+       concept_code_1,
+       vocabulary_id_1,
+       relationship_id,
+       valid_start_date,
+       valid_end_date,
+       invalid_reason,
+       concept_code_2,
+       vocabulary_id_2
+FROM concept_relationship_manual_backup_2023_02_08
+;
+
+
+--Detect codes not existing as naaccr values
+SELECT *
+FROM concept_relationship_manual_refresh
+WHERE concept_code_1 not in (SELECT concept_code from concept where concept_class_id='NAACCR Value')
+;
+
+--CHeck that NAACCR Values are not target for other codes
+SELECT *
```

Comment:
@vladkorsik To be moved to the "checks' file

Reply:
done